### PR TITLE
rename to hasOwn / accessible-object-hasownproperty

### PR DESCRIPTION
**Pull request by jamiebuilds**
This can be merged once the repo has been transferred to https://github.com/tc39

### CODE REVIEW

**Review by bnb — APPROVED** [review]

**Comment by ljharb** [comment]
Looks like it's transferred!  you'll need to update the domain in the "about" section on the repo homepage as well

**Comment by zloirock** [comment]
You forgot to update your polyfill.

File: polyfill.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 if (!Object.has) {
```
Comment (zloirock) [inline polyfill.js]: Also here.

File: polyfill.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
   var hasOwnProperty = Object.prototype.hasOwnProperty
 
-  Object.defineProperty(Object, "has", {
+  Object.defineProperty(Object, "hasOwn", {
     value: function(object, property) {
       return hasOwnProperty.call(object, property)
```
Comment (ljharb) [inline polyfill.js]:
```suggestion
      return hasOwnProperty.call(Object(object), property)
```
to reflect the updated exception ordering.

**Review by ljharb — APPROVED** [review]

File: polyfill.js (hunk 2 above)
Comment (zloirock) [reply polyfill.js]: It will kill an error on `null` / `undefined` and it's worse than the order of errors, in this case required an additional check.

Comment (zloirock) [reply polyfill.js]: @jamiebuilds ^^^

Comment (jamiebuilds) [reply polyfill.js]: 👀

**Pull request merged**